inst/stan/functions/generated_quantities.stan

```diff
@@ -14,7 +14,7 @@ int[] report_rng(vector reports, real[] rep_phi, int model_type) {
   int sampled_reports[t];
   if (model_type) {
     for (s in 1:t) {
-      sampled_reports[s] = neg_binomial_2_rng(reports[s] > 1e8 ? 1e8 : reports[s], rep_phi[model_type]);
+      sampled_reports[s] = neg_binomial_2_rng(reports[s] > 1e7 ? 1e7 : reports[s], rep_phi[model_type]);
```

Comment:
It'd be cleaner to define the upper and lower bounds globally so that it'll be consistent everywhere and could be easily tuned. I'm talking about `small = 1e-5` and `large = 1e8`, for example, or whatever a global variables will be named.

Reply:
All global constants could be moved to a header file.